sandy/njoy.py

```diff
@@ -855,7 +927,14 @@ def _groupr_input(endfin, pendfin, gendfout, mat,
         pk = len(ep) - 1
         text += [f"{pk} /"]
         text += [" ".join(map("{:.5e}".format, ep)) + " /"]
-    text += ["3/"]
+    if kwargs.get("mfcov", 1) == 40:
+        text += ["10/"]
+    else:
+        text += ["3/"]
+    if kwargs.get("mfcov", 1) == 34:
```

Comment:
These conditions should not be exclusive...we might want  both mubar  AND chi AND mf=10